pkg/interface/chat/src/js/components/lib/chat-input.js

```diff
@@ -176,27 +175,27 @@ export class ChatInput extends Component {
     this.bindShortcuts();
     
     return (
-      <div className="pa3 cf flex black bt b--black-30" style={{ flexGrow: 1 }}>
-        <div className="fl" style={{
-          marginTop: 4,
-          flexBasis: 32,
-          height: 36
-        }}>
-          <Sigil ship={window.ship} size={32} />
+      <div className="pa3 cf flex black bt b--gray4" style={{ flexGrow: 1 }}>
+        <div
+          className="fl"
+          style={{
+            marginTop: 4,
+            flexBasis: 24,
+            height: 24
+          }}
+        >
+          <Sigil ship={window.ship} size={24} color="#4330FC" />
         </div>
         <div className="fr h-100 flex" style={{ flexGrow: 1 }}>
           <textarea
-            className={'ml2 mt2 mr2 bn'}
-            style={{ flexGrow: 1, height: 40, paddingTop: 3, resize: 'none' }}
+            className={"pl3 bn"}
+            style={{ flexGrow: 1, height: 28, paddingTop: 6, resize: "none" }}
+            autoCapitalize="none"
             ref={this.textareaRef}
             placeholder={props.placeholder}
             value={state.message}
             onChange={this.messageChange}
-            autoFocus={true}
```

Comment:
Does this work?

https://stackoverflow.com/questions/15591661/how-to-focus-an-input-field-only-when-on-a-desktop-browser

Reply:
yes_meme.png

Added